src/Data/Textile/Query.elm

```diff
@@ -56,6 +57,15 @@ type alias MaterialQuery =
     }
 
 
+materialWithId : Material.Id -> Split -> Maybe Spinning -> Maybe Country.Code -> MaterialQuery
+materialWithId id share spinning country =
+    {  country = country
```

Comment:
Nit: formatting; please ensure running `elm-format` before committing (note: we may want to automate this in a pre-commit hook but that should be another PR)